tests/functional-homepage-centurion/add-footer-throughout.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { test, expect } from '@playwright/test';
+import { DashboardsIndexPage } from '../pages/dashboards-index.page';
+import { Footer } from '../pages/application-frame.page';
+
+test.describe('Homepage For Centurion - Footer Testing', () => {
```

Comment:
Can you use `IoT Application` instead of `Centurion` for naming?

Reply:
Done.